backup/__init__.py

```diff
@@ -0,0 +1,53 @@
+import dbbackup
```

Comment:
Never put anything in __init__.py files unless it's an import that makes something available to users importing your module. In case of django apps, just don't put anything there.

http://axialcorps.com/2013/08/29/5-simple-rules-for-building-great-python-packages/